theme/helpers.ftl

```diff
@@ -1062,13 +1062,18 @@
   [/#if]
 [/#macro]
 
-[#macro permHeader]
+[#macro permHeader showAdminLock=false]
   <header>
     <div class="perm-banner">
       <div class="banner-content">
         <a href="${PERM_DOMAIN}">
           <img class="banner-logo" src="${PERM_DOMAIN}/app/assets/img/logo/sideways_logo.png">
         </a>
+        [#if showAdminLock]
+          <a href="/admin/" title="Administrative Login">
+            <i class="fa fa-lock" style="font-size: 18px; color: #fff; opacity: 0.5;"></i>
```

Comment:
Is `fa-lock` a CSS class provided by FusionAuth?

Can we give it a bit more padding on the right? It looks a bit too close to the edge when the horizontal resolution is below 850px or so.

Reply:
`fa` and `fa-lock` is a Font Awesome  class I believe! It is weirdly coincidental that both have the same initials now that I think about it.

I will add more padding to this though!

Reply:
Oh ha, that makes sense!

Thank you!